package/native-package/package.json

```
@@ -17,7 +17,6 @@
     "@react-native-camera-roll/camera-roll": ">=5.0.0",
     "@react-native-community/netinfo": ">=2.0.7",
     "@react-native-clipboard/clipboard": "^1.11.1",
-    "@stream-io/flat-list-mvcp": "^0.10.3",
     "react-native": ">=0.60.0",
```

```suggestion
    "react-native": ">=0.72.0",
```

you missed a very important change here! and this should be done in core

also in expo , the relevant expo version of 0.72 and above must be set

---

also we should update the compatibility table in the docs

---

I was planning to do this in a separate PR since it will be easy for me to track and update the changes.